pkg/api/validation/validation.go

```diff
@@ -2844,10 +2799,16 @@ func ValidateNodeUpdate(node, oldNode *api.Node) field.ErrorList {
 	// Clear status
 	oldNode.Status = node.Status
 
+	// update taints
```

Comment:
> that doesn't follow from the fact that Tolerations are in the Spec;)

pod spec has been immutable since 1.0 with exactly two exceptions: images and activeDeadlineSeconds:

```
	if !apiequality.Semantic.DeepEqual(mungedPod.Spec, oldPod.Spec) {
		//TODO: Pinpoint the specific field that causes the invalid error after we have strategic merge diff
		allErrs = append(allErrs, field.Forbidden(specPath, "pod updates may not change fields other than `containers[*].image` or `spec.activeDeadlineSeconds`"))
	}
```